File: 1512-number-of-good-pairs/1512-number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
		int siz = nums.length;
		int count = 0;
		for (int i = 0; i < siz; i++)
		{
			for (int j = i+1; j < siz; j++)
			{
				if (nums[i] == nums[j])
					count++;
			}
		}
		return count;
	}
	
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums: List[int]) -> int:
        dic, count = dict(), 0
        for i in nums:
            if dic.get(i):
                count += dic.get(i)
            dic[i] = 1 + dic.get(i, 0)
        return count